src/backend/wiki_game_info/protondb/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { ProtonDBInfo, ProtonCompatibility } from 'common/types'
+import axios, { AxiosError } from 'axios'
+import { logDebug, logError, LogPrefix } from 'backend/logger/logger'
+
+export async function getInfoFromProtonDB(
+  steamID: string
+): Promise<ProtonCompatibility | null> {
+  if (steamID === '') {
+    logDebug('No SteamID, not getting ProtonDB info')
+    return null
+  }
+  const url = `https://www.protondb.com/api/v1/reports/summaries/${steamID}.json`
+
+  const response = await axios
+    .get<ProtonDBInfo>(url, { headers: {} })
+    .catch((error: AxiosError) => {
+      logError(
+        [
+          `Was not able to get ProtonDB data for ${steamID}`,
+          error.response?.data.error_description
+        ],
+        LogPrefix.ExtraGameInfo
+      )
+      if (error.response?.status === 404) {
+        return null
+      }
+      throw new Error('connection error', { cause: error })
```

Comment:
Don't throw anything. Just Log a message and return null. With this you complete Break heroic in offline mode